src/Adapter/EntityMapper.php

```diff
@@ -32,11 +32,6 @@
 use ObjectModel;
 use Shop;
 
-/**
```

Comment:
Why?

Reply:
Ping @jolelievre

Reply:
Because the comment is wrong : it's used and so not deprecated